### PR TITLE
Create an indexes markdown file to document all our Mongo indexes

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

- Adds all indexes listed from release 6.8.0 to 6.16.2 in the `INDEXES.md` file at the root of the repo

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## How do I test this PR?

Read and confirm contents of `INDEXES.md`
 
## How do we deploy this PR?

No special considerations.

### CODE REVIEW

**Review by nbachman-vox — APPROVED** [review]
This is a great idea, though i would like to see a doc somewhere of all the indexes we are using with info about what they address, for future reviews also 😸

**Comment by tessalt** [comment]
@nick-funk can you include an update to the pull request template that adds a checkbox for adding any new indexes to this file?

**Comment by nick-funk** [comment]
> @nick-funk can you include an update to the pull request template that adds a checkbox for adding any new indexes to this file?

yup! Will do!

**Comment by nick-funk** [comment]
> This is a great idea, though i would like to see a doc somewhere of all the indexes we are using with info about what they address, for future reviews also 😸

I was considering doing this, and I have decided I shall. I've added descriptions under the two dated indexes as I know what those are for. The other "existing indexes" I do not know off the top of my head why they exist, so they will remain anonymous for now.

However, this will get the ball rolling and ensure all future indexes are given a proper description of what they do in this file.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ In this section, note any new environment variables or feature flags introduced.
 
 -->
 
+## If any indexes were added, were they added to `INDEXES.md`?
+
+<!--
+
+In this section, check the `INDEXES.md` at the root of the repo and make sure you have added and commited any index changes necessary for this PR to be deployed. If you added any, indicated `Yes`in this section and list the index entries you added here.
+
+-->
```
Comment (nick-funk) [inline .github/PULL_REQUEST_TEMPLATE.md]: @tessalt, I have updated the PR template here with this `INDEXES.md` reminder. It's right under the env var and feature flags part. That way all the deployment "gotchas" are in one place together on the PR descriptions.

**Review by kabeaty — APPROVED** [review]

**Review by tessalt — APPROVED** [review]

**Pull request merged**